### ISSUE TITLE
Instance closed unexpectedly with code "3221225781" and signal "null"

### ISSUE THREAD

**Issue by 0xSwego** [nodkz/mongodb-memory-server#744]
## Versions

- NodeJS: 18.13.0
- mongodb-memory-server: 8.11.4
- mongodb: 6.0.3
- mongoose: 6.8.0
- system: Windows

package: mongo-memory-server 

## What is the Problem?

I get the error `Instance closed unexpectedly with code "3221225781" and signal "null"` when I try to execute
```JavaScript
await MongoMemoryServer.create();
```

This only happens on my new laptop (lg gram 16Z90O) while on my old one and on the ones from my colleagues everything work as expected.

## Debug Output

<details>
<summary>Debug Output</summary>

``` 
MongoMS:MongoMemoryServer create: Called .create() method +0ms
  MongoMS:MongoMemoryServer Mongo[unknown]: start: Called .start() method +0ms
  MongoMS:MongoMemoryServer Mongo[unknown]: _startUpInstance: Called MongoMemoryServer._startUpInstance() method +0ms
  MongoMS:MongoMemoryServer Mongo[unknown]: getStartOptions: forceSamePort: false +1ms
  MongoMS:MongoMemoryServer Mongo[unknown]: _startUpInstance: Creating new MongoDB instance with options: {
  instance: {
    port: 52599,
    dbName: '',
    ip: '127.0.0.1',
    storageEngine: 'ephemeralForTest',
    replSet: undefined,
    dbPath: '...\\Temp\\mongo-mem--28632-1e53Wydj5zrW',
    tmpDir: {
      name: '...\\Temp\\mongo-mem--28632-1e53Wydj5zrW',
      removeCallback: [Function: _cleanupCallback]
    },
    keyfileLocation: undefined,
    launchTimeout: undefined,
    args: undefined,
    auth: false
  },
  binary: undefined,
  spawn: undefined
} +2ms
  MongoMS:MongoInstance create: Called .create() method +0ms
  MongoMS:MongoInstance Mongo[52599]: start +1ms
  MongoMS:MongoBinary getPath +0ms
  MongoMS:DryMongoBinary generateOptions +0ms
  MongoMS:DryMongoBinary generateDownloadPath: Generating Download Path, preferGlobal: "true" +1ms
  MongoMS:DryMongoBinary generatePaths {
  version: '6.0.3',
  downloadDir: '',
  os: { os: 'win32' },
  platform: 'win32',
  arch: 'x64',
  systemBinary: ''
} +0ms
  
 ...

  MongoMS:DryMongoBinary generateDownloadPath: Found binary in resolveConfig (DOWNLOAD_DIR): "...\.cache\mongodb-binaries\mongod-x64-win32-6.0.3.exe" +1ms
  MongoMS:DryMongoBinary locateBinary: running generateDownloadPath +0ms
  MongoMS:DryMongoBinary generateDownloadPath: Generating Download Path, preferGlobal: "true" +1ms
  MongoMS:DryMongoBinary generatePaths {
  version: '6.0.3',
  downloadDir: '...\\.cache\\mongodb-binaries',
  os: { os: 'win32' },
  platform: 'win32',
  arch: 'x64',
  systemBinary: ''
} +0ms
  MongoMS:DryMongoBinary getBinaryName +1ms
  MongoMS:DryMongoBinary combineBinaryName +0ms
  MongoMS:DryMongoBinary combineBinaryName +1ms
  MongoMS:DryMongoBinary generatePaths: resolveConfigValue is not empty +0ms
  MongoMS:DryMongoBinary combineBinaryName +0ms
  MongoMS:DryMongoBinary combineBinaryName +0ms
  MongoMS:DryMongoBinary generateDownloadPath: Paths: {
  legacyHomeCache: '...\\.cache\\mongodb-binaries\\mongod-x64-win32-6.0.3.exe',
    at MongoInstance.closeHandler (...\node_modules\mongodb-memory-server-core\src\util\MongoInstance.ts:543:52)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at maybeClose (node:internal/child_process:1091:16)    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:513:28)
    at Socket.emit (node:domain:489:12)
    at Pipe.<anonymous> (node:net:320:12)
  MongoMS:utils Mongo[52599] killProcess: killerProcess: got exit signal, Code: null, Signal: SIGINT +11ms
  MongoMS:MongoInstance Mongo[52599]: stop: Instance Finished Shutdown +11ms
```

</details>

## Do you know *why* it happenes?

No

**Comment by hasezoey** [comment]
> Instance closed unexpectedly with code "3221225781" and signal "null"

try to install [vcredist](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170), [download latest or x86_64](https://aka.ms/vs/17/release/vc_redist.x64.exe)

**Comment by 0xSwego** [comment]
@hasezoey yeah it fixed it! Thanks!
